README.md

```diff
@@ -13,6 +13,26 @@ protocol written in [Go](http://golang.org/) initially based on [gogopherd](http
 
     go get -u github.com/peterhellberg/gopher/cmd/gopherd
 
+## Compile
+
+	go build -o gopher.local.native cmd/gopherd/main.go
```

Comment:
Instead of having a Compile section, the Installation section should be updated to use `go install github.com/peterhellberg/gopher/cmd/gopherd@latest` instead of `go get -u github.com/peterhellberg/gopher/cmd/gopherd`

Reply:
well, I don't want to install at this point, I want a local compile. The primary reason is that I can't compile go on my web server, so I am only interested in generating a binary that I can scp up. Installing would put the files in ~/go right? I don't mind documenting an install command.